app/controllers/accounts/apps/evolution_apis_controller.rb

```diff
@@ -0,0 +1,54 @@
+class Accounts::Apps::EvolutionApisController < InternalController
+  before_action :set_evolution_api, only: %i[ edit pair_qr_code]
+
+  def new
+    @evolution_api = Apps::EvolutionApi.new
+  end
+
+  def create
+    result = Accounts::Apps::EvolutionApis::Create.call(current_user, evolution_api_params)
+    @evolution_api = result[result.keys.first]
+    respond_to do |format|
+      if result.key?(:ok)
+        format.html { redirect_to account_apps_evolution_api_pair_qr_code_path(current_user.account, @evolution_api.id) }
+        format.json { render :create, status: :created }
+      else
+        @evolution_api = result[:error]
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @evolution_api[:error].errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def index
+    @evolution_apis = current_user.account.apps_evolution_apis.order(updated_at: :desc)
+    @pagy, @evolution_apis = pagy(@evolution_apis)
+  end
+
+  def edit
+  end
+
+  def pair_qr_code
+    # respond_to do |format|
+    #   if result.key?(:ok)
+    #     @whatsapp = result[:ok][:wp_connect]
+    #     @qr_code = result[:ok][:qr_code]
+
+    #     format.html { render :pair_qr_code, status: :ok }
+    #     format.json { render :pair_qr_code, status: :ok }
+    #   else
+    #     puts(result.inspect)
+    #     format.html { redirect_to account_apps_wpp_connects_path(current_user.account), status: :unprocessable_entity }
+    #   end
+    # end
+  end
+
+  private
+    def set_evolution_api
+      @evolution_api = Apps::EvolutionApi.find(params['evolution_api_id'])
```

Comment:
```suggestion
      @evolution_api = Apps::EvolutionApi.find(params[:id])
```

Será que assim já não funciona?

Reply:
Infelizmente não 😢 
![image](https://github.com/douglara/woofed-crm/assets/45243355/8662977b-5e6d-40d7-a1ad-ecde7c21f38c)

Reply:
Para a rota do pair_qr_code n, mas eu acabei fazendo essa alteração e na action do pair_qr_code eu coloquei esse codigo. Ficou assim: 
```ruby
  def pair_qr_code
    @evolution_api = current_user.account.apps_evolution_apis.find(params['evolution_api_id'])
  end

  private
    def set_evolution_api
      @evolution_api = current_user.account.apps_evolution_apis.find(params[:id])
    end
```